node/cmd/guardiand/adminclient.go

```diff
@@ -408,3 +417,43 @@ func runChainGovernorResetReleaseTimer(cmd *cobra.Command, args []string) {
 
 	fmt.Println(resp.Response)
 }
+
+func runPurgePythNetVaas(cmd *cobra.Command, args []string) {
+	daysOld, err := strconv.Atoi(args[0])
+	if err != nil {
```

Comment:
This handles the case where someone puts in "logonly" and not the DAYS_OLD argument?